SB2018/2018-03-15.md

```diff
@@ -0,0 +1,37 @@
+---
+to: All Teams
+subject: Tech day Location Change - This Saturday 17th March
+---
+
+Hi all,
+
+We've got another tech day this Saturday (17th March), However it's in a different location to usual!
+
+Tech days are a great opportunity for you to get robotics advice and help with technical issues from our volunteers. We may also have a tour of some university facilities as well.
+
+## Location
+The location is different to any previous tech days! It will be held inside the £4.5m electronics teaching labs inside [Building 59](http://data.southampton.ac.uk/building/59.html) at the University of Southampton's Highfield Campus:
+
+> Building 59\
+> University of Southampton\
+> University Road\
+> Southampton\
+> Hampshire\
+> SO17 1BJ
+
+There's a map attached showing where you should go to be let in. Its also on [google maps][maps-link]
+
+The day will run from 10:00am until 5:00pm, however feel free to come and go as you please.
+
+## Food
+Food on campus is scarce, so we recommend either ordering in food, or bringing something with you. There are a few corner-shops located nearby however. This time the tech day is happening right next door to pret a manger and costa so it should be easier to grab a coffee or an expensive lunch.
+
+## Electronics
+As expected, you'll need to bring your robot to work on, in whatever state it may be. We also recommend bringing at least 1 laptop with you to work on. Laptops can be provided, however this needs to be arranged with us in advance.
+
+## Supervision
+Team members must be accompanied by any adult who is able to take responsibility for all of them. (this would ideally be a teacher, but it can be a parent if all the other parents agree).
+
+If you're planning on attending, be sure to let us know, so we know to expect you. If you're unable to make this tech day, there's not long to wait. Our next scheduled tech day is the 17th Feb!
```

Comment:
Date for next tech day is wrong here. Also, perhaps remove the 'be sure to let us know' stuff, it's a bit late for that ;)

Reply:
fixed in